Sources/Commands/Error.swift

```diff
@@ -151,7 +151,10 @@ private func _handle(_ error: Any) {
         }
         stream <<< result.arguments.map{$0.shellEscaped()}.joined(separator: " ")
         print(error: stream.bytes.asString!)
-        
+
+    case Process.Error.missingExecutableProgram(let program):
+        print(error: "Unable to find an executable \(program).")
```

Comment:
looking at the rest of this file, it's pretty mixed - there's a fair bit of initial capitalization, although much fewer punctuation ending the errors. I removed the punctuation, but left the capitalization since it seemed to fit with more other lines.